chart/kubeapps/templates/dashboard/configmap.yaml

```diff
@@ -70,7 +70,7 @@ data:
       "kubeappsCluster": {{ include "kubeapps.kubeappsCluster" . | quote }},
       "kubeappsNamespace": {{ .Release.Namespace | quote }},
       "globalReposNamespace": {{ include "kubeapps.globalReposNamespace" . | quote }},
```

Comment:
This `globalReposNamespace` flag relates only to direct Helm plugin. Shouldn't we also move it to value e.g. `kubeappsapis.pluginConfig.core.packages.v1alpha1.globalPackagingNamespace` ?